pkg/cmd/scan.go

```diff
@@ -128,6 +128,13 @@ func NewScanCmd() *cobra.Command {
 		"Use those HTTP headers to query the provided URL.\n"+
 			"Only used with tfstate+http(s) backend for now.\n",
 	)
+	fl.StringVarP(&opts.BackendOptions.TerraformCloudToken,
+		"terraform-cloud-token",
```

Comment:
The standard abbreviation is `TFC`, so, `TFCLOUD` is coining a new one. I think it would be best to use what HashiCorp uses themselves, i.e. 'TF_API_TOKEN`.

Reply:
Thanks for pointing that out @nikolay, you're right. Also [in the docs](https://docs.driftctl.com/next/usage/cmd/scan-usage#http--terraform-cloud) we were already using the `TFC_TOKEN` env var for Terraform Cloud.